### DISCUSSION TITLE
override yearly budget with monthly budget

### DISCUSSION THREAD

**Discussion by learingmmex** [moneymanagerex/moneymanagerex#5247] [Q&A]
hi, i have been waiting for this option "override yearly budget with monthly budget" but unfortunately i did not know how to use it.

i created budget for 2023, then under this year i have budgets for the 12 months. when i am changing the budget for example for one month it is not reflected in the year budget.

i am not sure if i am using this feature correctly, would anyone help me understand how it can be used.

note: i am using version 1.6.0

thanks in advance

**Comment by whalley** [comment]
With this you first set the budget for the year and if the year is that predictable you can leave it there. However if you want to adjust any budget on a month-by-month basis you can override what would have been used by the yearly definition and that budget will be used for the month rather than the yearly amount.

You will see the effect of this in the budget reports.
